packages/api/tests/lambdas/test-internal-reconciliation-report.js

```diff
@@ -396,7 +400,7 @@ test.serial('internalRecReportForGranules handles generated granules with custom
       ...fakeGranule,
       granule: fakeGranule,
       executionUrl: fakeGranule.execution,
-      workflowStartTime: Date.now(),
+      createdAt: Date.now(),
```

Comment:
What's up with this change? We're not deriving the `createdAt` from the `workflowStarTime` so we can set it directly?

Reply:
This test was using generateGranuleApiRecord to generate a 'granuleApiRecord' and then translating it for write.    It needs to have a consistent start time - this test was using the default param (workflowStartTime) instead of passing it in with the granule originally, however given the changes this param no longer makes sense.  

Removed: 6ead93b5ee